packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -26,6 +26,7 @@ import { ModuleUtil } from '../util/ModuleUtil';
 
 export async function importNodesToEnvironment(environmentRegistryEntry: FabricEnvironmentRegistryEntry, fromAddEnvironment: boolean = false, createMethod?: string): Promise<boolean> {
     const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
+    const text: string = createMethod !== UserInputUtil.ADD_ENVIRONMENT_FROM_OPS_TOOLS ? 'import' : 'filter';
```

Comment:
Maybe this variable should be named better? So we know what text actually is?
Something like createMethodString?